static/css/left_sidebar.css

```diff
@@ -0,0 +1,227 @@
+.sidebar-left {
+  position: fixed;
+  background-color: #000000;
+  top: 0;
+  left: 0;
+  width: 20%;
+  height: 100%;
+}
+
+.card1 {
+  position: relative;
+  border-radius: 6px;
+  padding: 2rem 1rem;
+  color: #aaa;
+  overflow: hidden;
+  height: 570px;
```

Comment:
This is very bad and buggy. If text is bigger than 570px you can't see it. It happens on every page on the site.